runtime/doc/autocmd.txt

```diff
@@ -758,6 +758,15 @@ LspNotify			See |LspNotify|
 LspProgress			See |LspProgress|
 LspRequest			See |LspRequest|
 LspTokenUpdate			See |LspTokenUpdate|
+							*MarkSet*
+MarkSet				Just after a |mark| is set. The |event-data|
+				is a table with the following fields:
+
+				- mark: the character of the set mark. (ex:
+				  'a')
```

Comment:
Use common names where possible. `name` seems to fit here?

Note: also want to keep the door open for extending this to new marks in the future, maybe even user-defined marks. And possibly extmarks (which were loosely intended, as their name implies, to be a superset of "regular marks").

I don't see anything in your design that puts that at risk, but mentioning it in case you think of anything.